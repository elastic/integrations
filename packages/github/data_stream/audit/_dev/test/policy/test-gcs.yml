input: gcs
vars:
data_stream:
  vars:
    project_id: gcs-project
    number_of_workers: 3
    poll: true
    poll_interval: 15s
    service_account_key: "{\"type\":\"service_account\",\"project_id\":\"fake-gcs-project\"}"
    buckets: |
      - name: testbucket
    preserve_original_event: true
    preserve_duplicate_custom_fields: true
