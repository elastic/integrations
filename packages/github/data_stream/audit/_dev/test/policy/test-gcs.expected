inputs:
    - data_stream:
        namespace: ep
      meta:
        package:
            name: github
      name: test-gcs-github
      streams:
        - auth.credentials_json.account_key: ${SECRET_0}
          buckets:
            - name: testbucket
          data_stream:
            dataset: github.audit
            type: logs
          file_selectors: null
          max_workers: 3
          poll: true
          poll_interval: 15s
          project_id: gcs-project
          publisher_pipeline.disable_host: true
          tags:
            - preserve_original_event
            - preserve_duplicate_custom_fields
            - forwarded
            - github.audit
      type: gcs
      use_output: default
output_permissions:
    default:
        _elastic_agent_checks:
            cluster:
                - monitor
        _elastic_agent_monitoring:
            indices: []
        uuid-for-permissions-on-related-indices:
            indices:
                - names:
                    - logs-github.audit-ep
                  privileges:
                    - auto_configure
                    - create_doc
secret_references:
    - {}
